pyvista/plotting/plotting.py

```diff
@@ -570,6 +570,41 @@ def export_gltf(self, filename, inline_data=True, rotate_scene=True, save_normal
         for array in renamed_arrays:
             array.SetName('Normals')
 
+    def export_vrml(self, filename):
+        """Export the current rendering scene as a VRML file.
+
+        See https://vtk.org/doc/nightly/html/classvtkVRMLExporter.html
+        for limitations regarding the exporter.
+
+        Parameters
+        ----------
+        filename : str
+            Filename to export the scene to.  A filename extension of
+            ``'wrl'`` will be added.
```

Comment:
Is the extension added even if it already exists? If so, we might want to add that as a note (or find a way of disabling it if we specify an extension).

Reply:
I'm using the same style as the `export_vtkjs` comment, but it's not correct. Thanks for pointing that out.
```suggestion
            Filename to export the scene to.
```